CHANGELOG.md

```diff
@@ -19,6 +19,20 @@ All notable changes to this project will be documented in this file.
     * #### Added
     * #### Changed
         * `nidmm.Session()` no longer takes id_query parameter
+        * Option string can now be a python dictionary instead of a string. It will be converted as needed
+            * Valid options:
+                * RangeCheck (bool)
```

Comment:
We should document only the 'pythonic' spellings for these; even if things will work with the driver spellings.
Alternatively, we could support only the original spellings which allows us to remove knowledge* of valid options from nimi-python, which I think would be a good thing.

* with the exception of DriverSetup / driver_setup

I also don't think it's a good idea to document in such detail in changeling. What we should do is
* Document in the right place
* Link to the issue that got fixed
* In that issue we could document what we did as well